src/njoy21/input/POWR.hpp

```diff
@@ -0,0 +1,36 @@
+class POWR {
+public:
+
+using Card1 = WIMSR::Card1;
+#include "njoy21/input/POWR/Card2.hpp"
+
+#include "njoy21/input/POWR/Lib1.hpp"
+#include "njoy21/input/POWR/Lib2.hpp"
+#include "njoy21/input/POWR/Lib3.hpp"
+
+#include "njoy21/input/POWR/src/readLib1.hpp"
+#include "njoy21/input/POWR/src/readLib2.hpp"
+#include "njoy21/input/POWR/src/readLib3.hpp"
+
+Card1 card1;
+Card2 card2;
+optional< std::vector< Lib1 > > lib1;
```

Comment:
Since one can only have one lib type, perhaps this should be:

`std::vector< Variant< Lib1, Lib2, Lib3 > > lib.

Then we can have a single `readlib` function:

```
std::vector< Variant< ... > >
readlib( const int lib ){

if( lib == 1){
  std::vector< Lib1 > libs;
  ...
else if( lib == 2 ){
  std::vector< Lib2 > libs;
...

  return libs;
}
```

I think something like that will work.